plugins/by-name/lsp-signature/default.nix

```diff
@@ -40,7 +40,7 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
       Will show two lines of comment/doc(if there are more than two lines in the doc, will be truncated).
 
       Set to zero if you DO NOT want any API comments be shown.
-      This setting only take effect in insert mode, it does not affect signature help in nornal mode.
+      This setting only take effect in insert mode, it does not affect signature help in normal mode.
```

Comment:
Unrelated change: is this in a separate commit?
Or was this required by the pre-commit hook? (Is so I don't mind having it in one commit)

Reply:
Sorry, out pre-commit hook prevented my change from committing because of this.